openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -267,13 +269,65 @@ def _does_name_change_require_verification(user, old_name, new_name):
     """
     If name change requires verification, do not update it through this API.
     """
+
+    user_profile = UserProfile.objects.get(user=user)
+    profile_meta = user_profile.get_meta()
+    name_changes = 0
+    if 'old_names' in profile_meta:
+        name_changes = len(profile_meta['old_names'])
+
     return (
         is_verified_name_enabled()
-        and old_name != new_name
-        and len(get_certificates_for_user(user.username)) > 0
+        and (
```

Comment:
If the learner doesn't have any certificates, I believe they shouldn't be blocked from changing their name in any case

```suggestion
    and len(get_certificates_for_user(user.username)) > 0
    and (
        _edits_are_invalid(old_name, new_name)
        or name_changes >= 2
    )
```